website/static/js/fangorn.js

```diff
@@ -1031,9 +1031,17 @@ function _removeEvent (event, items, col) {
 
     // If there is only one item being deleted, don't complicate the issue:
     if(items.length === 1) {
+        var detail = 'This action is irreversible.';
+        if (window.contextVars.node.preprintFileId === items[0].data.path.replace('/', '')) {
+            // title = 'Delete the primary preprint file "' + items[0].data.name + '"?';
+            detail = [
+                m('p', 'This is the primary file for a preprint.'),
+                m('p', m('strong', 'Deleting this file will remove your preprint from circulation.'))
+            ];
+        }
         if(items[0].kind !== 'folder'){
             var mithrilContentSingle = m('div', [
-                m('p', 'This action is irreversible.')
+                m('p', detail)
```

Comment:
It might be nice to preserve the `This action is irreversible.` message for preprints too, but not a blocker